### ISSUE TITLE
ED patients that were transferred to ICU

### ISSUE THREAD

**Issue by rriccilopes** [MIT-LCP/mimic-code#1707]
### Prerequisites

* [X] Put an X between the brackets on this line if you have done all of the following:
    * Checked the online documentation: https://mimic.mit.edu/
    * Checked that your issue isn't already addressed: https://github.com/MIT-LCP/mimic-code/issues?utf8=%E2%9C%93&q=

### Description
I would like to do an study only with the ED stays that were transferred to the ICU. I tried to link the `edstays` with `icustays` using `stay_id`, but it returned no rows. I assume they are different ids, although they have the same name. I understand that I would have to use the `subject_id`, but then if the patient has multiple ED stays I wouldn't be sure which ED stay ended with ICU admission. Is there a way to link the ED stay with the ICU stay, even if the patient has multiple stays? Perhaps I'm missing something? Thanks!

**Comment by alistairewj** [comment]
Yeah I somewhat regret this naming convention. `stay_id` uniquely refers to an ED stay or an ICU stay, but not both. You can use the `hadm_id` from edstays to link to the subsequent hospitalization. Then, you can link that to icustays. ED stays should be 1:1 with `hadm_id`, but since a hospitalization can have multiple ICU stays, there may be more than one ICU `stay_id` for a given `hadm_id`.
